packages/spacer/src/Spacer.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { ThemeContext } from "@hig/themes";
+import { AVAILABLE_SPACINGS } from "@hig/theme-data-poc/build/index.es";
```

Comment:
If I point to just `"@hig/theme-data-poc"` the `tinycolor` causes an error in the enzyme testing